sdk/testing/src/main/java/io/opentelemetry/sdk/testing/assertj/OpenTelemetryAssertions.java

```diff
@@ -26,5 +26,5 @@ public static SpanDataAssert assertThat(SpanData spanData) {
     return new SpanDataAssert(spanData);
   }
 
-  protected OpenTelemetryAssertions() {}
+  private OpenTelemetryAssertions() {}
```

Comment:
I feel like there must have been a good reason for this to be `protected` explicitly. Is this being extended somewhere like the instrumentation project.

Reply:
Not used https://github.com/open-telemetry/opentelemetry-java-instrumentation/search?q=OpenTelemetryAssertions

Reply:
If it turns out there is a good reason for this (maybe in one of the custom agent distributions?), we can very easily open it back up. 👍